argocd/overlays/moc-infra/projects/tremor-demo.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
Can you replace all instance if `<team-name>` with `tremor-demo` ? 

This should be made clearer in the doc I admit, will make a note of it.

Reply:
sorry, must have missed that. I meant to replace :P